config/components/manager/manager.yaml

```diff
@@ -49,4 +49,9 @@ spec:
             cpu: 500m
             memory: 512Mi
       serviceAccountName: controller-manager
+      tolerations:
+        - key: kueue.x-k8s.io/nodegroup
+          effect: NoSchedule
+          operator: Exists
+          value: "true"
```

Comment:
```suggestion
```
?

Reply:
thanks for taking a look.
Updated to become "kueue.x-k8s.io/nodegroup Equal controller".

Also considering to have a more generic key like "component" instead of "kueue.x-k8s.io/nodegroup". With this more generic key we can have the controller of other systems (maybe like ray, jobset etc.) in the same cluster have their controllers to be schedule on the same node pool as well. What do you think?